queries.sql

```diff
@@ -78,4 +78,55 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) AS avg_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+/*Day three the used queries*/
+
+-- Melody's animals
+
+SELECT animals.name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+-- list of pokemon animals
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+-- list of all animals and owns
+
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+--animal numbers per spacies
+
+SELECT species.name, COUNT(animals.id)
+FROM species
+JOIN animals ON species.id = animals.species_id
+GROUP BY species.name;
+
+--list of Jennifer's digimons
+
+SELECT animals.name
+FROM animals
+JOIN owners ON animals.owner_id = owners.id
+JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
+
+--list of animals owned by Dean Winchester that haven't tried to escape
+
+SELECT * FROM animals WHERE owner_id = 5 AND escape_attempts = 0;
+
```

Comment:
- [ ] (REQUIRED) The screenshot of this query was not attached to the pull request description. Kindly attach it. 👍 😎 
- [OPTIONAL] This query is hardcoded because you know the id of the owner. Imagine a case where you don't know the id of the owner but you have his name. It would be better to use a more reliable query that searches for the owner's name and the number of escape_attempt from the animals that the person owns. 💻 🌵 

#### Hint:
```SQL
 SELECT animals.* FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   WHERE owners.full_name = 'Dean Winchester'
   AND animals.escape_attempts = 0;
```

Reply:
I added it but I will add it again ;)

Reply:
@otmanTR , that was the only screenshot missing because you changed your query to the hard-coded one and probably forgot to add it. 

![image](https://user-images.githubusercontent.com/77106962/230238283-b122b9f8-f65b-4b28-a944-0621e2723b5d.png)

I a glad you added the correct one. 👏